packages/components/datepicker/calendar-header.component.ts

```diff
@@ -12,6 +12,13 @@ import { DateAdapter } from '@koobiq/components/core';
 const defaultMinYear = 1900;
 const defaultMaxYear = 2099;
 
+type monthName = {
```

Comment:
```suggestion
type MonthName = {
```
предлагаю типы/интерфейсы с большой буквы писать
в будущем добавлю [это правило](https://typescript-eslint.io/rules/naming-convention/)